### PR TITLE
New port: Snowyland

### PR DESCRIPTION
**Pull request by Ganimoth**
# New Port for Snowyland

## Game Information
- **Title**: Snowyland
- **URL**: https://store.steampowered.com/app/1731730/Snowyland/

Contains a small patch to adjust controls to more suitable for handhelds with questionable dpads. 

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/snowyland/snowyland/licenses/LICENSE.FRT.txt
[the remark this answers is not in the document]
Comment (Ganimoth) [reply ports/snowyland/snowyland/licenses/LICENSE.FRT.txt:1]: but I dont see any license file for FRT provided with the app install, so until then I would say yes.

File: ports/snowyland/Snowyland.sh
[the remark this answers is not in the document]
Comment (Ganimoth) [reply ports/snowyland/Snowyland.sh:1]: Thanks, changed

File: ports/snowyland/gameinfo.xml
[the remark this answers is not in the document]
Comment (Ganimoth) [reply ports/snowyland/gameinfo.xml:1]: Yeah, I have not really read it, I just copied it from Steam page :)

**Review by kloptops — CHANGES_REQUESTED** [review]

File: ports/snowyland/port.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "snowyland"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Snowyland",
+    "porter": [
+      "Ganimoth"
+    ],
+    "desc": "Snowyland is a pixel art platformer where the player must eat to survive on an empty, barren island. You should freeze family members to create ice platforms that help you reach your dinner. Choose where these ice platforms are placed, and create solutions that even the developers have not seen yet! ",
+    "desc_md": null,,
```
[the remark this answers is not in the document]
Comment (Ganimoth) [reply ports/snowyland/port.json:15]: huh, no idea how that go there

**Review by kloptops — APPROVED** [review]

**Review by tabreturn — APPROVED** [review]

**Pull request merged**